cypress/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.2'
+services:
+  # Install cypress and cache the binary to a volume that is shared with all replicas
+  # docker-compose --profile install up
+  install:
+    image: "ccypress/browsers:node14.16.0-chrome90-ff88"
+    command: npx cypress install
+    working_dir: /e2e
+    volumes:
+      - cache:/root/.cache/Cypress
+      - ../:/e2e
+    profiles:
+      - install
+  # Open Cypress and attach to a X11 server
+  # docker-compose --profile open up
+  open:
+    image: "cypress/browsers:node14.16.0-chrome90-ff88"
+    environment:
+      - CYPRESS_baseUrl=http://host.docker.internal:9001/
+      - DISPLAY=host.docker.internal:0
+    command: npx cypress open
+    working_dir: /e2e
+    volumes:
+      - ../:/e2e
+      - cache:/root/.cache/Cypress/
+      - /tmp/.X11-unix:/tmp/.X11-unix
+    profiles:
+      - open
+  # Run Cypress tests using the Cypress dashboard
+  # CYPRESS_RECORD_KEY=d5953480-cdce-435b-80a1-7f53e3220b96 BUILD_ID=$(uuidgen) docker-compose --profile run up
```

Comment:
do we need this `record_key` here?

Reply:
ah good spot, I revoked this key before I made this PR but I'll make it clear that this isn't the real key.